src/adapter/src/catalog/builtin.rs

```diff
@@ -1765,6 +1765,19 @@ pub static MZ_SOURCE_STATISTICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSourc
         .with_column("bytes_received", ScalarType::UInt64.nullable(false)),
     is_retained_metrics_relation: true,
 });
+pub static MZ_SINK_STATISTICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
+    name: "mz_sink_statistics",
+    schema: MZ_INTERNAL_SCHEMA,
+    data_source: Some(IntrospectionType::StorageSinkStatistics),
+    desc: RelationDesc::empty()
+        .with_column("id", ScalarType::String.nullable(false))
+        .with_column("worker_id", ScalarType::UInt64.nullable(false))
+        .with_column("messages_staged", ScalarType::UInt64.nullable(false))
```

Comment:
the source metrics have these grouped differently: I think here it should also be:
 - `messages_staged`
 - `messages_committed`
 - `bytes_staged`
 - `bytes_committed`